tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -165,3 +167,248 @@ def test_upgrade_brokers(self, from_version, to_version):
         node = self.driver.node
         node.account.ssh("grep ALL-RECORDS-DELIVERED %s" % self.driver.STDOUT_FILE, allow_fail=False)
         self.processor1.node.account.ssh_capture("grep SMOKE-TEST-CLIENT-CLOSED %s" % self.processor1.STDOUT_FILE, allow_fail=False)
+
+    @parametrize(old_version=str(LATEST_0_10_1), new_version=str(LATEST_0_10_2))
+    @parametrize(old_version=str(LATEST_0_10_1), new_version=str(LATEST_0_11_0))
+    @parametrize(old_version=str(LATEST_0_10_1), new_version=str(DEV_VERSION))
+    @parametrize(old_version=str(LATEST_0_10_2), new_version=str(LATEST_0_11_0))
+    @parametrize(old_version=str(LATEST_0_10_2), new_version=str(DEV_VERSION))
+    @parametrize(old_version=str(LATEST_0_11_0), new_version=str(DEV_VERSION))
+    def test_simple_upgrade(self, old_version, new_version):
```

Comment:
I'd suggest update the existing test directly in this PR.

Reply:
Can you elaborate? Should we remove `test_upgrade_downgrade_streams` or merge this with `test_upgrade_downgrade_streams` ?

Reply:
This is for your comment above:

> Note: 1.0 was the first branch that did contain upgrade system tests already -- I just added the new once -- we can consider to consolidate them in this PR already. LMKWYT.

I'd suggest we consolidate them in this PR already, same for 1.1